### DISCUSSION TITLE
Custom Sample Banks

### DISCUSSION THREAD

**Discussion by TheFantasticLoki** [ppy/osu#16287] [Ideas]
I feel giving us the ability to define custom samplesets would be quite useful. We can already manually type custom bank names in lazer it just doesn't save if it's not normal soft or drum. This would solve a very annoying issue with map compilations and such for hitsounding and doesn't sound that hard to implement in both lazer and a hotfix for stable.

**Comment by peppy** [comment]
Do you mean in addition to the functionality already offered on stable? As in the ability to name sample sets rather than having a numeric assignment?

**Reply by TheFantasticLoki** [reply]
yes that's what I am proposing

**Comment by l4ssc** [comment]
This would make keysounding far more accessible as it doesn't take up space from other sets
